README.md

```diff
@@ -1,4 +1,17 @@
-# Tau Labs
+# Tau Labs - Brain FPV Flight Controller
```

Comment:
Nope :-P
Also, I don't know if we should really keep all the brain stuff below since we don't detail other targets.